changelog/11473.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:changes
```

Comment:
It looks like this should be `change`? https://github.com/hashicorp/vault/blob/5aa9047ff282b4fd9fcb865ddbe5c0bac2588896/changelog/changelog.tmpl#L17-L17
```suggestion
```release-note:change
```

Reply:
I've seen others with `changes` so I used that, but thanks for pointing out the `tmpl` file. I made the suggested change, thanks!